x11-packages/rnote/build.sh

```diff
@@ -0,0 +1,67 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/flxzt/rnote"
+TERMUX_PKG_DESCRIPTION="An infinite canvas vector-based drawing application for handwritten notes"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="0.11.0"
+TERMUX_PKG_SRCURL="https://github.com/flxzt/rnote/archive/v${TERMUX_PKG_VERSION}/v${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256=b133d4331963d3c09d3a7477f60fc4c5072471dcbf459379a593ca1724164af4
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_DEPENDS="gdk-pixbuf, gettext, glib, graphene, gtk4, hicolor-icon-theme, libadwaita, libcairo, pipewire, pango, poppler"
+TERMUX_PKG_BUILD_DEPENDS="libiconv"
+
+__fetch_gettext_rs() {
+    # This version is relative to gettext-sys provided by the gettext-rs crate
+	local _version="0.21.4"
+	local _url="https://github.com/gettext-rs/gettext-rs/archive/refs/tags/gettext-sys-$_version.tar.gz"
+	local _path="$TERMUX_PKG_CACHEDIR/gettext-v$_version.tar.gz"
+	termux_download "$_url" "$_path" SKIP_CHECKSUM
```

Comment:
Since we're downloading from a Github release tarball, please include the checksum.
```suggestion
	_sha256="211773408ab61880b94a0ea680785fc21fad307cd42594d547cf5a056627fcda"
	termux_download "$_url" "$_path" "$_sha256"
```